### PR TITLE
chore: add 'import-bundle' package, the receiving end of bundle-source

### LINKED ISSUE #739: add "import-bundle" package
I'm going to add a package named `import-bundle`, which is the evaluation-time receiver for the data artifacts created by `bundle-source`. The `import-bundle` package will have an export named `importBundle`, that is an async function which takes a bundle (exactly as emitted by `bundleSource()`) and yields a module namespace. It will have an option for extra endowments.

For now, this will be used by SwingSet, to load the kernel bundle and all static vats. Eventually it will help factor out this code in zoe (although this might instead turn into a dynamic vat creation call):

https://github.com/Agoric/agoric-sdk/blob/d39c827866cab9ad8e9d114353a8ff5feb000843/packages/zoe/src/evalContractCode.js#L12-L23

`import-bundle` will depend upon `ses-adapter` (#724), so it should be usable from outside a SwingSet/SES environment.

### PR DESCRIPTION
**Pull request by warner**
closes #739

Note, for unit tests, `install-ses.js` disables some SES security features,
one because of a bug in SES (Agoric/SES-shim#237), and another because
`rollup` uses Math.random for reasons that are not clear. We'll need to fix
or find workarounds for both: eventually this test should work without those
two patches.

### CODE REVIEW

**Comment by warner** [comment]
@michaelfig could you take a look at my unit test? The `transforms` test (in `do-import.js`) is failing, it complains that `added` is not defined, but the transform function I apply ought to be adding it to the endowments. Am I using it right?

Note that to see the actual error, you'll have to hack some code in `node_modules/` that tries to modify `Error.stackTraceLimit`: see https://github.com/Agoric/SES-shim/issues/247 for details. But maybe you can spot my mistake without really needing to run the test.

This isn't ready for review yet, I want to wait for a new SES first, in the hopes that I can remove the `noTameRegExp: true` flag.

**Comment by michaelfig** [comment]
I'm stymied as to why this doesn't work.  Can you narrow it down to a miniature test that we can add to the SES repo?

**Comment by warner** [comment]
I've added a narrowed test case to https://github.com/Agoric/SES-shim/issues/248 , thanks!

**Comment by warner** [comment]
Incidentally, the test failure appears to be a collision between:

1: `tap`'s habit of using `stack-utils` to `stack.captureString` when the subtest is created (presumeably to show later, just in case something in the subtest fails)
2: `stack-utils`'s habit of overriding the global `Error.stackTraceLimit` while calling `Error.captureStackTrace` (and then restoring it back again)
3: the SES tamed `Error` object's refusal to allow `stackTraceLimit` to be set

The test is not actually failing.. any `tap`-based subtest would fail the same way, if `lockdown()` had previously been called, and the SES environment thus created would throw an error when trying to set `stackTraceLimit`. I might be able to work around this by omitting the subtest, but I *think* I need one to accomodate an async function-under-test.

File: packages/import-bundle/test/do-import.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import bundleSource from '@agoric/bundle-source';
+import tap from 'tap';
+import { importBundle } from '../src/index.js';
+
+// function transform(rewriterState) {
```
Comment (erights) [inline packages/import-bundle/test/do-import.js]: Note https://github.com/Agoric/SES-shim/pull/255

Comment (warner) [reply packages/import-bundle/test/do-import.js]: oops, yep, I'll rewrite the test to match the actual API.

**Comment by warner** [comment]
Ok, this is ready for review. Some notes:

* it doesn't work at all outside of SES: we need https://github.com/Agoric/SES-shim/issues/224 (`Compartment` outside of SES) to make `ses-adapter` work without SES, plus the import-time-vs-later issue described in https://github.com/Agoric/agoric-sdk/issues/739#issuecomment-605496877 prevents me from using `ses-adapter` anyways
* The SES tests use `noTameMath`, because `bundle-source` uses `rollup` which uses `Math.random` for weird hash table reasons. This will be necessary for actual use too, which is a drag: it would be nice if the "Start Compartment" could use Math.random but other Compartments could not, but of course they all share primordials.
* @michaelfig points out that the way this provides endowments won't work for our metering needs, so that's still a work in progress

**Review by michaelfig — APPROVED** [review]
I love it!  Really great work, Brian!

**Pull request merged**